integration-tests/src/test/resources/setupenv.sh

```diff
@@ -66,6 +66,9 @@ function pull_tag_images {
 
   docker pull wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest
   docker tag wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest store/oracle/weblogic:12.2.1.3
+  
+  docker pull wlsldi-v2.docker.oraclecorp.com/weblogic-for-rc2:12.2.1.3.0
```

Comment:
Let's update this to use the recently pushed store/oracle/weblogic:12.2.1.3, which is already patched.  Also, please don't use the "rc2" element in the name.

Reply:
I tried running the tests against the latest  store/oracle/weblogic:12.2.1.3, which gets pulled using 
docker pull wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest
  docker tag wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest store/oracle/weblogic:12.2.1.3

Managed server logs contains these errors, the image doesn't seem to be a patched one.
<Jan 18, 2019 8:54:16,214 PM GMT> <Error> <Kernel> <BEA-000802> <ExecuteRequest failed
 java.lang.AssertionError: LocalGroup should atleast have the local server!.
java.lang.AssertionError: LocalGroup should atleast have the local server!
	at weblogic.cluster.messaging.internal.GroupImpl.send(GroupImpl.java:176)
	at weblogic.cluster.messaging.internal.server.UnicastFragmentSocket.send(UnicastFragmentSocket.java:97)
	at weblogic.cluster.FragmentSocketWrapper.send(FragmentSocketWrapper.java:84)
	at weblogic.cluster.UnicastSender.send(UnicastSender.java:53)
	at weblogic.cluster.UnicastSender.send(UnicastSender.java:21)
	Truncated. see log file for complete stacktrace

Reply:
I don't think wlsldi-v2.docker.oraclecorp.com/store-weblogic-12.2.1.3:latest has been updated with the patched image. 
Can you pull store/oracel/weblogic:12.2.1.3 directly from the docker store, instead of pulling from the internal registry? Alternatively, you need to ask Brian to update the image in the internal registry.